README.md

```diff
@@ -22,30 +26,36 @@ dependencies {
     compile "com.github.kinecosystem:kin-core-android:<latest release>"
 }
 ```
-For latest release version go to https://github.com/kinecosystem/kin-core-android/releases
 
-## Usage
+For latest release version go to [https://github.com/kinecosystem/kin-core-android/releases](https://github.com/kinecosystem/kin-core-android/releases).
+
+The main repository is at [github.com/kinecosystem/kin-core-android](https://github.com/kinecosystem/kin-core-android).
+
+## Get Started
+
 ### Connecting to a service provider
-Create a new `KinClient` with two arguments: an android `Context` and a `ServiceProvider`. 
 
-A `ServiceProvider` provides details of how to access the Stellar horizon end point.
-The example below creates a `ServiceProvider` that will be used to connect to the main (production) Stellar 
-network
-```java
-ServiceProvider horizonProvider =  
-    new ServiceProvider("https://horizon.stellar.org", ServiceProvider.NETWORK_ID_MAIN);
-KinClient kinClient = new KinClient(context, horizonProvider);
-```
+Create a new `KinClient` using a builder, set an `Environment` object that provides details of how to access the kin blockchain end point, Environment provides the predefined 
+`Environment.TEST` and `Environment.PRODUCTION`.  
+An optional parameter is `storeKey` which can be used to create a multiple accounts data set,
+ each different `storeKey` will have a separate data, an example use-case - store multiple users accounts separately.
```

Comment:
I think the storeKey parameter needs more explanation?
If I get this right the string I pass as storeKey will effect the account I'll get when calling gtAccount()?

Reply:
yes, but we can make it maybe better by moving from array to key based store inside Kin Client itself, let's discuss